lib/changes_are_logged.rb

```diff
@@ -22,16 +22,26 @@ def log_it
         @changes_logged = {}
         save_change_log
       elsif self.changed? || !@change_comments.blank?
-        @changes_logged = self.changes
+        exclude_list = self.class.log_changes_options.fetch(:exclude, []).map(&:to_s)
```

Comment:
propose using `:except`, which is a more-or-less standard option on a number of built-in rails methods